### PR TITLE
Parallels tools install fix when enabled in Vagrantfile

### PR DESCRIPTION
**Pull request by earljon**

### CODE REVIEW

File: scripts/vm-guest-tools.ps1 [hunk 1]
```diff
@@ -78,4 +78,17 @@ if ("$env:PACKER_BUILDER_TYPE" -eq "virtualbox-iso") {
...
+  Write-Output "Using Parallels Desktop"
+  # This is required when using Parallels Desktop
+  # Currently the installation of Parallels Tools is broken if you're installing in Windows Server Core
+  # You will encounter an error in printui.dll module not found message.
+  # This feature will add the required libraries for printing services.
+
+  # Parallels Tools is required for sync_folder to work.
+  # Vagrantfile config: v.update_guest_tools = true will be installed with no issues
+  Write-Output "Installing Print Server service"
+  Install-WindowsFeature -Name Print-Server
```
Comment (StefanScherer) [inline scripts/vm-guest-tools.ps1:91]: Sorry, this command won't work on Windows 10, and this provision script is used for all Windows templates.

```
PS C:\Windows\system32> Install-WindowsFeature -Name Print-Server
Install-WindowsFeature : The term 'Install-WindowsFeature' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At line:1 char:1
+ Install-WindowsFeature -Name Print-Server
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Install-WindowsFeature:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Maybe something similar to https://github.com/StefanScherer/packer-windows/blob/main/scripts/debloat-windows.ps1#L18 could help to run this only on Windows Server.

**Review by StefanScherer — CHANGES_REQUESTED** [review]

**Review by StefanScherer — CHANGES_REQUESTED** [review]
Thank you for your contribution. It needs some enhancement to make the provision script work with all Packer templates.

File: scripts/vm-guest-tools.ps1 (hunk 1 above)
Comment (earljon) [reply scripts/vm-guest-tools.ps1:91]: Ah, right, Im concentrated to work this on my workflow (WIndows Server) without checking out other versions of Windows.

Let me fix and test this @StefanScherer. Thanks!

**Comment by earljon** [comment]
Hi @StefanScherer , I've updated the code to include condition to install only on Windows Server types

**Review by StefanScherer — APPROVED** [review]
@earljon Thank you! Now it should work on Windows 10 as well.
LGTM

**Pull request merged**